sql/core/src/test/scala/org/apache/spark/sql/streaming/TransformWithStateSuite.scala

```diff
@@ -195,6 +179,20 @@ class TransformWithStateSuite extends StateStoreMetricsTest
     }
   }
 
+  test("Use statefulProcessor without transformWithState - handle should be absent") {
+    val processor = new RunningCountStatefulProcessor()
+    val ex = intercept[Exception] {
+      processor.getHandle
+    }
+    checkError(
+      ex.asInstanceOf[SparkRuntimeException],
+      errorClass = "STATE_STORE_HANDLE_NOT_INITIALIZED",
+      parameters = Map.empty
+    )
+    assert(ex.getMessage.contains("The handle has not been initialized" +
```

Comment:
Could you remove the assert. The error text can be modified to tech writer, or it can be translated to other languages. If we expect some text in tests, this won't be possible w/o rewriting such tests.

Reply:
Done